xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -921,7 +916,7 @@
  The default ClearType font smoothing value is 1400, but it can be any value from 1000 to 2200. Depending on the display device and the user's sensitivity to colors, a higher or lower contrast value may improve readability.  
   
 > [!NOTE]
->  This property is supported only on Windows XP and the Windows Server 2003 family.  
+> This property is supported only on Windows XP and the Windows Server 2003 family.  
```

Comment:
Should this say "Unsupported"? I presume the docs are now targeting only supported platforms, since we're removing "notes" completely below.